### DISCUSSION TITLE
v5.2.0

### DISCUSSION THREAD

**Discussion by EtienneLem** [missive/emoji-mart#653] [Announcements]
I recognize that v5 has been pretty much unusable in a real case scenario, sorry about that.
**v5.2.0** should be much more stable. 

## 🎉 New features
- [`@emoji-mart/react`](https://github.com/missive/emoji-mart#react) package, for seamless usage in React

## 🔨 Improvements
- Allow to set an id for custom categories [#650]
- Compile code to support wider range of browsers [#619]
- Update element when its attributes change [#643]
- Basic support for RTL languages
- Provide CommonJS, ESM, and browser builds

## ⚙️ Fixes
- Props are not memoized anymore [#624]
- SSR / NextJS [#575]
- Fix picker being duplicated in React [#617]
- Fix certain events and observers leaking

## 🗺 New languages
- Arabic
- Dutch
- Japanese
- Persian
- Spanish
- Ukrainian

## 🧹 Chores
* Bump terser [#648]

## 🧑‍🎓 New Contributors
* @breadadams made their first contribution in https://github.com/missive/emoji-mart/pull/608
* @the-fukui made their first contribution in https://github.com/missive/emoji-mart/pull/618
* @hariksone made their first contribution in https://github.com/missive/emoji-mart/pull/625
* @arnoldligtvoet made their first contribution in https://github.com/missive/emoji-mart/pull/627
* @kiruba56 made their first contribution in https://github.com/missive/emoji-mart/pull/590
* @AbhayVAshokan made their first contribution in https://github.com/missive/emoji-mart/pull/619
* @abolfazl-talaei made their first contribution in https://github.com/missive/emoji-mart/pull/628
* @FnrDev made their first contribution in https://github.com/missive/emoji-mart/pull/630
* @Clebal made their first contribution in https://github.com/missive/emoji-mart/pull/650

---
**Full Changelog**: https://github.com/missive/emoji-mart/compare/v5.1.0...v5.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/missive/emoji-mart/releases/tag/v5.2.0'>v5.2.0</a>.</em>

**Comment by LookLukeLemon** [comment]
CommonJS and SSR!!! 👏
